src/installer/installer.go

```diff
@@ -97,18 +97,12 @@ func (i *installer) InstallNode() error {
 	}
 
 	i.UpdateHostInstallProgress(models.HostStageInstalling, i.Config.Role)
-	ignitionFileName := i.Config.Role + ".ign"
+	ignitionFileName := i.Config.HostID + ".ign"
```

Comment:
Consider having the role in as part of the file name.

Reply:
I assume we don't have any plans to change the role or a host, right?

Reply:
Not an argument against the suggestion. Just curious as it might make this a bit strange.